node/node.go

```diff
@@ -1423,13 +1423,23 @@ func LoadStateFromDBOrGenesisDocProvider(
 
 // panics if failed to unmarshal bytes
 func loadGenesisDoc(db dbm.DB) (*types.GenesisDoc, error) {
-	b, err := db.Get(genesisDocKey)
+	var b []byte
+	iter, err := db.Iterator(append(genesisDocKey, byte(0)), append(genesisDocKey, byte(255)))
```

Comment:
nit: Please help me understand. Why do we read with every 255 bytes?

Reply:
Oh, it doesn't mean it want to read 255 bytes. In the past, if it was stored as a key called `genesisDoc`, now it is stored in this way: `genesisDoc0`,` genesisDoc1`, ... (here, the numbers of 0 and 1 are bytes, not ints). It means to do up to genesisDoc255.

Reply:
Oh, I got it! Thanks. Could you write down the its spec in the PR description?